src/content/docs/dropins/product-details/tutorials/gift-options.mdx

```diff
@@ -0,0 +1,256 @@
+---
+title: Add gift options to PDP pages
+description: Learn how to customize the PDP drop-in component to include gift options.
+---
+
+import Aside from '@components/Aside.astro';
+import Diagram from '@components/Diagram.astro';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+
+The [`GiftOptions` container](/dropins/cart/containers/gift-options/) allows you to add gift options, such as gift wrapping or personalized messages, to product detail pages. This feature enhances the shopping experience by enabling customers to select these options before adding products to their cart.
+
+The code examples provided below demonstrate the general approach to building custom integrations with the `GiftOptions` container.
```

Comment:
I guess we can slightly rephrase this since integration with PDP is just an example. Technically, GiftOptions container can be integrated on any page following same or similar approach (this is not bound to PDP).